1. **You**: find the error and fix it app.get('/test', (req: Request, res: Response, next: NextFunction) => {
	res.status(200).json({
		success: true,
		message: 'api is working',
	});
});
2. **You**: find the error and fix it app.get('/test', (req: Request, res: Response, next: NextFunction) => {
	res.status(200).json({
		success: true,
		message: 'api is working',
	});
});
3. **You**: find the error and fix it app.get('/test', (req: Request, res: Response, next: NextFunction) => {
	res.status(200).json({
		success: true,
		message: 'api is working',
	});
});
4. **You**: find the error and fix it app.get('/test', (req: Request, res: Response, next: NextFunction) => {
	res.status(200).json({
		success: true,
		message: 'api is working',
	});
});
5. **You**: find the error and fix it app.get('/test', (req: Request, res: Response, next: NextFunction) => {
	res.status(200).json({
		success: true,
		message: 'api is working',
	});
});
6. **You**: find the error and fix it app.get('/test', (req: Request, res: Response, next: NextFunction) => {
	res.status(200).json({
		success: true,
		message: 'api is working',
	});
});